### ISSUE TITLE
Installing on VS2017 15.9.13, no options available to run it

### ISSUE THREAD

**Issue by peterlukerow** [microsoft/dotnet-apiport#781]
Having installed it there is no option on the Analyze menu or on the context menu for the solution.
If try to install it again it says it's already installed.

On the Tools >> options dialog there is no section for the .NET portability analyzer.

**Comment by Lxiamail** [comment]
@peterlukerow  Thanks for reporting the issue. Just want to double check, after you install the VSIX of .NET Portability Analyzer, have you closed all the active VS processes to let the VSIX finish, then restart the VS process?

**Comment by peterlukerow** [comment]
I had one instance of VS 2017 running, in fact only 1 instance of any VS. 

- I closed it
- Went to task manager and checked for any processes begin with "v" - there were none
- Re-opened VS 2017, loaded my solution

After the above there are still no menu/context menu options.

HOWEVER - I've discovered the bug. I am running VS as a Windows admin user account. If I run VS as my standard user account then the options appear. I run as admin because I run my web app using my local IIS rather than IIS express so that I can test as close to the real thing as possible.
